cassandra-analytics-core/src/main/java/org/apache/cassandra/clients/Sidecar.java

```diff
@@ -210,6 +212,8 @@ public static final class ClientConfig
         public static final String CHUNK_BUFFER_SIZE_BYTES_KEY = "chunkBufferSizeBytes";
         public static final String MAX_POOL_SIZE_KEY = "maxPoolSize";
         public static final String TIMEOUT_SECONDS_KEY = "timeoutSeconds";
+        public static final String CACHE_COMPRESSION_METADATA_KEY = "cacheCompressionMetadata";
+        public static final String MAX_SIZE_CACHE_COMPRESSION_METADATA_KEY = "maxCacheCompressionMetadata";
```

Comment:
Can we add the units for this param?
```suggestion
        public static final String MAX_SIZE_CACHE_COMPRESSION_METADATA_KEY = "maxCacheCompressionMetadataMiB";
```